third_party/boringssl/BUILD.gn

```diff
@@ -177,3 +178,14 @@ static_library("crypto_full") {
     configs += [ "//starboard/build/config:speed" ]
   }
 }
+
+if (is_starboard) {
+  target(final_executable_type, "boringssl_tool") {
+    testonly = true
+    sources = tool_sources
```

Comment:
When I was building this I just put the sources directly inline here, but it doesn't really matter either way. Just seems weird to create a BUILD.generated.gni that isn't actually generated

Reply:
I copied it from trunk :) , i'll move inline to reduce surprises